src/Message.php

```diff
@@ -0,0 +1,315 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Mailer;
+
+use DateTimeImmutable;
+use DateTimeInterface;
+use Throwable;
+
+final class Message implements MessageInterface
```

Comment:
Need phpdoc and likely changes to README/docs.

Reply:
done